actix-http/src/body/body.rs

```diff
@@ -16,15 +16,16 @@ use super::{BodySize, BodyStream, MessageBody, MessageBodyMapErr, SizedStream};
 pub type Body = AnyBody;
 
 /// Represents various types of HTTP message body.
-pub enum AnyBody {
+#[derive(Clone)]
+pub enum AnyBody<B = BoxBody> {
     /// Empty response. `Content-Length` header is not set.
     None,
 
     /// Specific response body.
```

Comment:
This may better be worded as in memory response body.
Typically the generic body variant is for async streaming where the vaule is not able and/or desirable to be in memory as a whole. 
This would help them to construct the correct one.